include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -1138,22 +1138,26 @@ class CommonOperatorLoader : public ProtobufLoader {
     return Error::success();
   }
 
-  // Loads Less operator. Internally it's a cmpLT Node.
-  Error loadLess(const OpType &op, ArgumentDictionaryTy &dict) {
-    // Input Type.
+  // Loads compare (Equal, Less and Greater) operators.
+  Error loadCmpOps(llvm::StringRef typeName, const OpType &op) {
+    std::string opName = loadOperatorName(op);
     NodeValue xNV;
     ASSIGN_VALUE_OR_RETURN_ERR(xNV, getNodeValueByName(op.input(0)));
     NodeValue yNV;
     ASSIGN_VALUE_OR_RETURN_ERR(yNV, getNodeValueByName(op.input(1)));
-
-    std::string opName = loadOperatorName(op);
-
-    auto *xNode = xNV.getNode();
-    auto *yNode = yNV.getNode();
-
-    Node *N = G_->createNodeWithBroadcast<CmpLTNode>(opName, /* axis */ -1,
-                                                     xNode, yNode);
-
+    auto *LHS = xNV.getNode();
+    auto *RHS = yNV.getNode();
```

Comment:
Is this necessary, to use the Node instead of the NodeValue? Most of the node creator APIs take NodeValues. Doing it like this could lead to bugs, since it means if `xNV`/`yNV` was not the first result of a multi-output Node then we will instead use the first output (this should be caught in debug mode via an assert, but still).